pkgs/development/python-modules/clingo/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3,
+  python3Packages,
```

These cannot be used in python-modules, since they would tie the package to a specific python version.

```suggestion
  buildPythonPackage,
  python,
  setuptools,
  cffi,
```